deploy.sh

```diff
@@ -1,6 +1,30 @@
 #!/usr/bin/env bash
-rm index.zip
-cd lambda
-zip -r -X ../index.zip *
-cd ..
-aws lambda update-function-code --function-name chemex-first-floor --zip-file fileb://index.zip
\ No newline at end of file
+
+if [ -z "$1" ]; then
+    echo "Provide a function name."
+    echo "Usage: deploy.sh <lambda-function-name>"
+    exit 1
+fi
+
+# Test AWS existence
+which aws 1>/dev/null 2>&1
+
+if [ "$?" -ne "0" ]; then
+    echo "Please install the AWS cli tools."
+    exit 2
+fi
+
+# Remove the zip file, if it exists
+rm super-chemex-bot.zip 2>/dev/null
+
+echo "Zipping the lambda/ directory..."
+cd lambda; zip -q -r -X ../super-chemex-bot.zip *; cd ..
+
+echo "Deploying to AWS..."
+aws lambda update-function-code --function-name $1 --zip-file fileb://super-chemex-bot.zip >/dev/null
+
+if [ "$?" -eq "255" ]; then
```

Comment:
what about other status codes? Maybe having it the other way around (checking for `0`) makes more sense?

Reply:
I could do that but there should be nothing but 255 or 0 being returned: https://docs.aws.amazon.com/cli/latest/topic/return-codes.html

Reply:
Hmm... okay. Still would prefer the other way around but yeah. 👍